templates/page.html

```diff
@@ -182,8 +182,8 @@ <h3>Select a REANA cluster...</h3>
         <h3>...or install your own</h3>
         <figure class="figure">
           <div class="command-line">
-            <div class="line comment"># install kubectl 1.14.0 and minikube 1.0.0</div>
-            <div class="line"><span class="prompt">$</span>sudo dpkg -i kubectl*.deb minikube*.deb</div>
+            <div class="line comment"># install kubectl 1.14.0, minikube 1.0.0 and helm 2.12.3</div>
```

Comment:
@roksys  what do you think about doing [as we do in Invenio](https://github.com/inveniosoftware/inveniosoftware.org/blob/4e7840274121cd624320d2688fd04265efacb132/templates/gettingstarted.html#L21), linking to the installation requirements? see the result https://inveniosoftware.org/#getstarted